ci/azure-jobs-linux.yml

```diff
@@ -0,0 +1,54 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
+  variables:
+    OS: 'linux'
+    ARCH: 'x86_64'
+  strategy:
+    matrix:
+      Ubuntu_1404_GCC:
+        IMAGE: librepcb/librepcb-dev:ubuntu-14.04-1
```

Comment:
Also here: Why Ubuntu 14.04? It is end of life since April 2019.

Reply:
Because 14.04 provides Qt 5.2 (and GCC 4.8), which LibrePCB still supports, and currently I see no reason to remove this support :wink:

Reply:
I'm still amazed by your dedication to support legacy systems :slightly_smiling_face: But I'd consider then supporting Debian Jessie which is still in LTS and has gcc 4.9 and Qt 5.3. But there is no point in bloating this PR with this.

Reply:
:+1: Adding more CI jobs makes sense anyway, but yes, this can be done later in a separate PR.